src/Pages/Login/index.js

```diff
@@ -1,7 +1,179 @@
 import React, { Component } from 'react';
+import { Link, withRouter } from 'react-router-dom';
+import { fetchPost } from '../../utils/fetches';
+import { REGEXP, validate } from '../../utils/regex';
+import './index.scss';
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { userId: '', userPw: '', loggedUser: {} };
+  }
+
+  validateInputData = (id, pw) => {
+    return (
+      validate(id, REGEXP.emailRegExp) && validate(pw, REGEXP.passwordRegExp)
+    );
+  };
+
+  handleInput = (e) => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+
+    const { userId, userPw } = this.state;
+
+    if (!this.validateInputData(userId, userPw)) return;
+
+    fetchPost('http://10.58.6.148:8000/users/login', {
```

Comment:
```suggestion
    fetchPost('${API}/users/login', {
```
여러 군데서 사용되고 자주 변경될 수 있는 ip 주소 등은 src/config.js 같은 경로로 분리해주어야 코드의 관리포인트가 줄어들어 편해집니다!

Reply:
반영했습니다.